config/initializers/security_headers.rb

```diff
@@ -1,13 +1,13 @@
-::SecureHeaders::Configuration.configure do |config|
-  config.hsts                   = {:max_age => 99, :include_subdomains => true}
+::SecureHeaders::Configuration.default do |config|
+  config.hsts                   = 'max_age=99; include_subdomains=true'
   config.x_frame_options        = 'DENY'
-  config.x_content_type_options = "nosniff"
-  config.x_xss_protection       = {:value => 1, :mode => false}
+  config.x_content_type_options = 'nosniff'
+  config.x_xss_protection       = 'value=1; mode=false'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.